File: backend/auth-file/internal/entity/member.go
package entity

import (
	"github.com/google/uuid"
)

type Member struct {
	Id       uuid.UUID `db:"id"`
	Email    string    `db:"email"`
	Password string    `db:"password"`
	Name     string    `db:"name"`
	Role     string    `db:"role"`
	GroupId  string    `db:"group_id"`
}

File: backend/fileserver/src/main/java/com/jungwook/fileserver/domain/Member.java
package com.jungwook.fileserver.domain;

import jakarta.persistence.*;
import lombok.*;
import org.hibernate.annotations.UuidGenerator;

import java.util.UUID;

@Entity
@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Member {
  @Id
  @UuidGenerator(style = UuidGenerator.Style.VERSION_7)
  private UUID id;

  @Column(columnDefinition = "TEXT", nullable = false)
  private String name;

  @Column(columnDefinition = "TEXT", nullable = false)
  private String email;

  @Column(columnDefinition = "TEXT", nullable = false)
  private String password;

  @Column(columnDefinition = "TEXT", nullable = false)
  private String role;

  @ManyToOne(fetch = FetchType.LAZY)
  @JoinColumn(
      name = "group_id",
      nullable = false
  )
  private Group group;
}
